File: typescript/7kyu/The_@_operator.ts
export const evaluate = (equation: string) => {
  const r = equation
    .split("@")
    .map(Number)
    .reduce((a, b) => a + b + (a - b) + a * b + (b === 0 ? NaN : (a / b) | 0));
  return isNaN(r) ? null : r;
};

File: python/7kyu/The_@_operator.py
def evaluate(equation):
    a, *numbers = map(int, equation.split(" @ "))
    
    for b in numbers:
        if b == 0:
            return None
        
        a = (a + b) + (a - b) + (a * b) + (a // b)
    
    return a